src/renderers/webgl/WebGLState.js

```diff
@@ -259,8 +259,8 @@ function WebGLState( gl, extensions, capabilities ) {
 			setOp: function ( stencilFail, stencilZFail, stencilZPass ) {
 
 				if ( currentStencilFail	 !== stencilFail 	||
-				     currentStencilZFail !== stencilZFail ||
-				     currentStencilZPass !== stencilZPass ) {
+					 currentStencilZFail !== stencilZFail ||
+					 currentStencilZPass !== stencilZPass ) {
```

Comment:
i think it would be better to leave that code as it is.

Reply:
> I think it would be better to leave that code as it is.

Screenshot in VSC.
| Before | After |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/23699926/89106426-536db580-d45c-11ea-8da0-75bd71f8a2bc.png) | ![image](https://user-images.githubusercontent.com/23699926/89106407-2c16e880-d45c-11ea-84cc-4ecb611fda60.png) |

Reply:
Try changing the tab size in VSC.

Reply:
> Try changing the tab size in VSC.

Sorry, I have no idea about how to change the tab size in VSC, please tell me how to do it.

Reply:
I found this one google:
https://stackoverflow.com/questions/29972396/how-can-i-customize-the-tab-to-space-conversion-factor

Reply:
> I found this one google:
> https://stackoverflow.com/questions/29972396/how-can-i-customize-the-tab-to-space-conversion-factor

Is that right?

| Tab size: 4 | Tab size: 2 |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/23699926/89107130-75b60200-d461-11ea-8bf8-00e701062447.png) | ![image](https://user-images.githubusercontent.com/23699926/89107154-a0a05600-d461-11ea-9275-c94a2ba1066e.png) |

Reply:
I don't know, I don't use VSC. I use Atom, and in Atom is a simple setting:

<img width="768" alt="Screen Shot 2020-08-02 at 12 31 17 AM" src="https://user-images.githubusercontent.com/97088/89111565-2936ea80-d492-11ea-818a-6615bc3ebf51.png">

Reply:
Anyway, don't worry about it. I'll merge this and revert these changes.